### ISSUE TITLE
FSE/FSE themes: Submenu creation, and submenu background and text color issue

### ISSUE THREAD

**Issue by vajrasar** [Automattic/wp-calypso#65407]
### Quick summary

On the Twenty Twenty Two (TT2) theme using the Page (Large Header) template we cannot create a submenu item. 

To create submenu via Site Editor you will need to change the page template > create the submenu > save the menu.

If you use the same submenu then in the Page (Large Header) the background and text/link color of the submenu always remains white.

### Steps to reproduce

1. Create a site with TT2 theme.
2. Assign the homepage to use the Page (Large Header) template
3. Open Site Editor > select the Page (Large Header) template
4. In the Header Block > try to create submenu using the steps explained [here](https://wordpress.com/support/full-site-editing/theme-blocks/navigation-block/#create-submenus)
5.  You cannot create the submenu in step 4, hence switch to another template like Page
6. Create new menu with submenu in the Page template > save the template (and hence the changes will save the menu too)
7. Go back to the Page (Large Header) template in the site editor
8. In Header > Navigation > use the menu saved in step 6 > Save changes
9. Check the site's homepage on how the submenu items show up

### What you expected to happen

1. It should be possible to create a menu with submenu on any page template of the site.
2. The submenu should follow the theme's color scheme

PS: The other pages of the site will show the menu and submenu in expected color scheme. 

### What actually happened

1. Cannot create a new menu with submenu using site editor when the Page (Large Header) template is selected.
2. The menu with submenu created on the Page (Large Header), created using steps 5-9, should show with site's expected color scheme, but instead it shows submenu background + text/links both in white color.

### Context

5358428-zen

### Simple, Atomic or both?

Simple, Atomic

### Theme-specific issue?

Twenty Twenty Two

### Browser, operating system and other notes

N/A

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

Yes, difficult to implement

### Workaround details

Custom CSS code.

**Comment by KirkwallDay** [comment]
Ran across an instance of this here: 5357738-zen

Gave this CSS as a workaround:

```
.wp-site-blocks .wp-block-navigation a {
color: inherit;
}
```

User would like to be notified when the bug is fixed so they can remove that code.

- [ ] 5357738-zen

While investigating the bug I found the menu item colors for the sub-menus are pulling the background color in CSS, screenshot here:
![Screen Shot 2022-07-08 at 9 05 46 AM](https://user-images.githubusercontent.com/49131029/178019630-9c157a5e-c53c-4171-bf79-057aab28e49d.png)

**Comment by Robertght** [comment]
📌 HOUSEKEEPING
- Labels ✅ 
- Priority ✅ 
- Replicable on Core - No

📌 SCRUBBING
- Tested on P2 ✅ 
- Tested on Simple ✅ 
- Tested on AT ✅ 
- Tested on Self-hosted ✅ 

📌 FINDINGS/SCREENSHOTS/VIDEO
- There are 2 issues here:

1. You cannot create a submenu within the Site editor. The + button doesn't is not clickable when trying to add a submenu item: 
![image](https://user-images.githubusercontent.com/6737345/178318584-278313ef-4530-4d1e-af04-7b8ab2ab45b9.png)

I tested this and it appears to happen only on .com hosted sites.

2. The colors of the submenus are not consistent with the editor choices. I found another reference here: https://github.com/Automattic/wp-calypso/issues/65419 and also, I noticed with Pendant with another user last week after the Gutenberg update their color choice for the navigation block was overridden by the global styles one.

Another reference here: https://github.com/Automattic/wp-calypso/issues/65419

cc @Automattic/cylon and @scruffian for further visibility on this as I'm not sure what's going on.
 

📌 ACTIONS
- Marked as Triaged for Quality Squad review

**Comment by jeyip** [comment]
@Addison-Stavlo, @edanzer, and I investigated this further. We looked at the Twenty Twenty Two theme and how it creates a default Page (Large Header) template with a navigation block. The navigation block is subsequently populated by a page list block.

<img width="1280" alt="Screen Shot 2022-07-20 at 9 45 09 AM" src="https://user-images.githubusercontent.com/5414230/180037821-5faabc4a-ab78-4db2-b9f3-cb131cb712ea.png">

The page list block doesn't appear to support submenu item creation. To do so, we'd have to convert the page list block to individual links, at which point submenu creation would be available.

![2022-07-20 09 50 18](https://user-images.githubusercontent.com/5414230/180038752-75e497d4-1c88-437b-9e81-02879e5d7b4f.gif)

To me, this process feels very unintuitive, but it _does_ look like expected behavior for now. Moving forward, we could look for an existing issue in the [core Gutenberg](https://github.com/WordPress/gutenberg) that describes this problem and continue the discussion there. If it doesn't exist, we could spin up a feature request in the Gutenberg repo ourselves. Either way, I believe we can close this particular Calypso issue. Would love to hear your thoughts on this @vajrasar
